### PR TITLE
feat(console): table empty state

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add component: table empty state.

## Linear Issue Reference

LOG-1970

## Testing

截屏2022-03-24 下午3 36 13

### CODE REVIEW

File: packages/console/src/components/Table/TableEmpty.module.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@use '@/scss/underscore' as _;
+
+.tableEmpty {
+  text-align: center;
+
+  .image {
+    text-align: center;
```
Comment (simeng-li) [inline packages/console/src/components/Table/TableEmpty.module.scss:7]: will text-align inherit from the parent node?

File: packages/console/src/components/Table/TableEmpty.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <tr>
+      <td colSpan={0}>
+        <div className={styles.tableEmpty}>
+          <div className={styles.image}>
+            <img src={EmptyImage} />
+          </div>
+          <div className={styles.title}>{title ?? t('admin_console.errors.empty')}</div>
```
Comment (simeng-li) [inline packages/console/src/components/Table/TableEmpty.tsx:24]: nit: {title && <div className={styles.title}>{ t('admin_console.errors.empty')}</div>} ?

File: packages/console/src/components/Table/TableEmpty.module.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+@use '@/scss/underscore' as _;
+
+.tableEmpty {
```
Comment (xiaoyijun) [inline packages/console/src/components/Table/TableEmpty.module.scss:3]: Missing a `padding-top`.
image

**Review by xiaoyijun — DISMISSED** [review]
Overall LGTM.

File: packages/console/src/components/Table/TableEmpty.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/Table/TableEmpty.module.scss:7]: ditto in `.title` & `.content`

File: packages/console/src/components/Table/TableEmpty.tsx (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/components/Table/TableEmpty.tsx:24]: Is a default title, `&&` may not work?

File: packages/console/src/components/Table/TableEmpty.module.scss (hunk 3 above)
Comment (wangsijie) [reply packages/console/src/components/Table/TableEmpty.module.scss:3]: add 4 units padding (table row's padding is 5 units), PTAL

File: packages/console/src/components/Table/TableEmpty.module.scss (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/components/Table/TableEmpty.module.scss]: updated

File: packages/core/src/routes/application.ts [hunk 4]
```diff
@@ -31,6 +31,8 @@ export default function applicationRoutes<T extends AuthedRouter>(router: T) {
     ctx.pagination.totalCount = count;
     ctx.body = applications;
 
+    ctx.body = [];
```
Comment (xiaoyijun) [inline packages/core/src/routes/application.ts:34]: Forget to delete this line?

**Review by xiaoyijun — APPROVED** [review]

**Pull request merged**